### DISCUSSION TITLE
Tooltip is breaking react tests

### DISCUSSION THREAD

**Discussion by brianhumphreys** [twilio-labs/paste#3444] [Q&A] [answered]
Does anyone know why simply adding a tooltip would break a react test?  The tooltip is imported from paste.  This is the error I am seeing:

```
.pnpm/jest-runner@29.6.2/node_modules/jest-runner/build/runTest.js:341:7),
      type: 'unhandled exception'
    }

      75 |     setDeliveredFilter: jest.fn()
      76 |   }
    > 77 |   const renderWithProps = (props: Props) => render(<PushAnalytics {...props} />)
         |                                                   ^
      78 |   const renderComponentFilteredByDelivered = () =>
      79 |     render(
      80 |       <PushAnalytics
```

What is the unhandled exception in this case?

I have attached a screen recording of what is happening.

https://github.com/twilio-labs/paste/assets/26912737/df9fa25e-ae95-4ccc-8d4a-8ba5d7405045

**Comment by serifluous** [comment]
Hey @brianhumphreys ! Let us know if the solution Seth shared in the evergreen-paste slack channel worked out.

Otherwise, can you provide us a full stack trace and reproduction?

**Reply by serifluous** [reply, marked as answer]
Marking as answered due to inactivity. Feel free to reply if you need more support!
